src/components/app-configure/AppUserLimits.vue

```diff
@@ -306,4 +360,17 @@ input[type="number"] {
 .usage .custom-select__trigger {
   padding: 20px;
 }
+.error-message {
+  font-family: var(--font-body);
+  font-weight: 400;
+  visibility: hidden;
+  font-size: 0.94em;
+  line-height: 1.5em;
```

Comment:
Line heights are unitless.

```suggestion
  line-height: 1.5;
```